src/lib/sign.ts

```diff
@@ -1,44 +0,0 @@
-import { kmsClient } from "./utils/kms/kmsClient";
-import { changeId } from "./utils/tools/changeId";
-
-export default async function sign(data: any, keyName: string): Promise<any> {
-  if (
-    !data ||
-    !keyName ||
-    !process.env.kmsProjectId ||
-    !process.env.signKeyVersion
-  ) {
-    console.log(data, keyName, process.env.kmsProjectId);
-    console.log(
-      "Please specify both data/keyName/process.env.kmsProjectId/process.env.signKeyVersion",
-    );
-    throw new Error(
-      "Please specify both data/keyName/process.env.kmsProjectId/process.env.signKeyVersion",
-    );
-  }
-
-  const safeId = changeId(keyName);
-
-  const fullKeyName = kmsClient.cryptoKeyVersionPath(
-    process.env.kmsProjectId,
-    "global",
-    safeId,
-    "sign",
-    process.env.signKeyVersion,
-  );
-
-  const uint8Array = new Uint8Array(Object.values(data));
```

Comment:
This won't produce the expected result if someone sends non-binary `data`, as `data` would then be a plain string, so potentially we are doing something similar to `new Uint8Array(['H', 'e', 'l', 'l', 'o'])`, which won't encode the string properly.